Makefile

```diff
@@ -38,9 +38,9 @@ ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 K8S_VERSION=1.19.0
 test: generate-installer fmt vet bundle
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
-	sed -i "s,#\!.*,#\!\/bin\/bash,g" ${ENVTEST_ASSETS_DIR}/setup-envtest.sh
-	sed -i "/pipefail/d" ${ENVTEST_ASSETS_DIR}/setup-envtest.sh
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh # this is a workaround while we don't upgrade the sdk
+	#sed -i "s,#\!.*,#\!\/bin\/bash,g" ${ENVTEST_ASSETS_DIR}/setup-envtest.sh
+	#sed -i "/pipefail/d" ${ENVTEST_ASSETS_DIR}/setup-envtest.sh
```

Comment:
Afraid to remove them completely? 😆

Reply:
Well, I didn't want to remove since it was created by the SDK, should cause less confusion when migrating to newer versions (i.e., the code block will be there, though commented out vs. not having the lines at all)

Reply:
Oh no, I added this to fix an error in their script. I believe I should've sent a PR instead. We can remove these lines.